app/app/layout.tsx

```diff
@@ -19,26 +19,41 @@ export default function RootLayout({
         <link rel="shortcut icon" href="/favicon.png" />
       </head>
       <body>
-        <div className={`fixed bottom-0 right-0 translate-x-[20%] translate-y-[20%]`}>
+        <div
+          className={`fixed bottom-0 right-0 translate-x-[20%] translate-y-[20%]`}
```

Comment:
should use twmerge

Reply:
We should, still didn't change that much on App, mostly focused on finishing the landing now, maybe @desamtralized can add it